ui/src/context/control.ts

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+import { UC_AUTH_API_PREFIX } from '../utils/constants';
+
+/**
+ * The singleton client instance with `baseURL`.
+ */
+export const CLIENT = axios.create({
```

Comment:
It looks like we are instantiating the client twice? Here and in the catalog.ts file